release/ray_release/command_runner/_prometheus_metrics.py

```diff
@@ -13,9 +13,29 @@
 
 DEFAULT_PROMETHEUS_HOST = "http://localhost:9090"
 PROMETHEUS_HOST_ENV_VAR = "RAY_PROMETHEUS_HOST"
+DEFAULT_PROMETHEUS_HEADERS = "{}"
+PROMETHEUS_HEADERS_ENV_VAR = "RAY_PROMETHEUS_HEADERS"
 RETRIES = 3
 
 
+# parse_prom_headers will make sure the input is in one of the following formats:
+# 1. {"H1": "V1", "H2": "V2"}
+# 2. [["H1", "V1"], ["H2", "V2"], ["H2", "V3"]]
+def parse_prom_headers(prometheus_headers):
```

Comment:
This file seems to be highly similar to `metrics_head.py`. What are the differences between them? Perhaps @alanwguo knows the difference.

Reply:
I have removed the change from the `_prometheus_metrics.py`. It seems we will never need this feature in the release procedures.